tests/integration/components/ember-table-test.js

```diff
@@ -205,12 +226,18 @@ for (let customHeader of customHeaderTests) {
 
 test('Test custom row', async function(assert) {
   await setupFullTable(this, {}, {}, DEFAULT_ROW_COLUMN_COUNT, 'custom-row');
-  assert.ok(find('tbody tr').className.indexOf('custom-row') >= 0, 'Table has custom row');
+
+  let tablePage = TablePage.create();
+
+  assert.ok(tablePage.body.rows.eq(0).containsClass('custom-row'), 'Table has custom row');
```

Comment:
To do this without a generic `containsClass` method, you would do:

```js
tablePage = TablePage.extend({
  body: {
    rows: collection({
      isCustomRow: hasClass('custom-row')
    })
  }
}).create();
```

In actual usage, this is most likely what people will be doing. If you'd like to create a generic version of the page for these tests, you could also add it in the extension. The point is that in general people will not have generic classes on tables, but specific classes.

Reply:
Hmm, it seems the body is overriden instead of being merged. Let's pair tomorrow.